diff --git a/BUILD.gn b/BUILD.gn
index c2c11c24ace..a8763df62d0 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -1514,6 +1514,10 @@ config("toolchain") {
       # warning.
       cflags += [ "-Wctad-maybe-unsupported" ]
     }
+
+    if (is_win) {
+      cflags += [ "-fansi-escape-codes" ]
+    }
   }
 
   if (is_clang || !is_win) {
@@ -3109,6 +3113,7 @@ v8_header_set("v8_headers") {
     "include/v8-wasm.h",
     "include/v8-weak-callback-info.h",
     "include/v8.h",
+    "include/wasm-v8-lowlevel.hh",
   ]
 
   sources += [
@@ -5797,6 +5802,7 @@ v8_source_set("v8_base_without_compiler") {
       "src/wasm/wasm-subtyping.cc",
       "src/wasm/well-known-imports.cc",
       "src/wasm/wrappers.cc",
+      "src/wasm-v8-lowlevel.cc",
     ]
     if (!is_official_build) {
       sources += [
